### ISSUE TITLE
[Medium] Chrome+JAWS: Functionalities associated with Pin button is not announced by Screen Reader

### ISSUE THREAD

**Issue by SumitDiyora** [Expensify/App#9440]
## Action Performed:
1. Using Chrome+JAWS, open URL: staging.new.expensify.com
2. Login with the credentials.
3. On the homepage, select any chat.
4. Navigate to the visually looking PIN button and select it located at the top right beside the 'Call' button.
5. Observe if the Screen reader is making any announcement.

## Expected Result:
After clicking on the visually looking Pin button or by selecting the Pin button, the Screen Reader should announce that the selected chat has been pinned or unpinned.

## Actual Result:
After Clicking or Selecting the visually looking Pin button, the Screen Reader is not announcing its status as it is pinned or unpinned.

## Workaround:
Yes

## Area issue was found in:
In the chats section beside the call button 

## Failed WCAG checkpoints
4.1.2

## User impact:
The Screen reader users will not be able to know the status of the Pin button, whether the chat has been pinned or unpinned, and thus they will not able to use this feature.

## Suggested resolution:
Ensure that the Screen reader announces the status of the chat (pinned/unpinned) when the user selects or clicks the Pin image button. Alternatively, usage of the aria-live attribute can announce the status as desired.

Refer:
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

## Platform:
- Web
- iOS
- Android
- Desktop App
- Mobile Web

Version Number: *v1.1.76-2*
Reproducible in staging?: *Yes*
Reproducible in production?: *Yes*
Issue reported by: *Sumit*

https://user-images.githubusercontent.com/98817464/173531571-6838d6dc-7098-4df8-a851-aca16ad157f5.mp4
